dbms/src/Flash/Pipeline/SpillExecutor.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
Consider naming it `SpillThreadPool` as the `Executor` name is occupied by the top-level entrance structure. Plus, it is a thread pool actually.

Reply:
updated.